src/tasks/minions/HunterActivity/hunterActivity.ts

```diff
@@ -143,6 +143,19 @@ export default class extends Task {
 				loot.add(itemID('Baby chinchompa'));
 			}
 		}
+		if (creature.name === 'Eastern ferret') {
+			const cl = user.cl();
+			const amountOfZippysGotten = cl.amount('Zippy');
```

Comment:
I really don't like this CL mechanic for non-limited items. 
It makes competing for mostdrops impossible for newcomers, and more importantly, significantly demotivates players from continuing the content after 1 or two drops. 

If you think the rate is too common, then make it rarer, but I disagree with doing this for non-limited items.